src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -977,7 +973,7 @@ public function offsetGet($option, bool $triggerDeprecation = true)
             }
 
             if (!\is_array($value)) {
-                throw new InvalidOptionsException(sprintf('The nested option "%s" with value %s is expected to be of type array, but is of type "%s".', $this->formatOptions([$option]), $this->formatValue($value), get_debug_type($value)));
+                throw new InvalidOptionsException(sprintf('The nested option "%s" with value "%s" is expected to be of type array, but is of type "%s".', $this->formatOptions([$option]), $this->formatValue($value), get_debug_type($value)));
```

Comment:
this change should be reverted. It is a false positive of the fabbot check because `formatValue` already puts quotes around string values.